js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -42,10 +43,10 @@ jest.mock('../../../../core/utils/scroll_rtl_behavior', () => () => testBehavior
 }].forEach(({ viewFunction, Scrollable }) => {
   describe('Scrollable', () => {
     describe('Render', () => {
-      it('should render scrollable content', () => {
+      it('should render scrollView content', () => {
         const scrollable = shallow(viewFunction({ props: { } } as any) as JSX.Element);
-        const scrollableContent = scrollable.find('.dx-scrollable-wrapper > .dx-scrollable-container > .dx-scrollable-content');
-        expect(scrollableContent.exists()).toBe(true);
+        const scrollViewContent = scrollable.find('.dx-scrollable-wrapper > .dx-scrollable-container > .dx-scrollable-content > .dx-scrollview-content');
```

Comment:
'should add scrolling classes by default' test should check 'dx-scrollable-wrapper'

Reply:
Yep, great idea.
Fixed https://github.com/DevExpress/DevExtreme/pull/15636/commits/3371f7a4122918b13a658b8a49751be74f06fa71